nsxt/resource_nsxt_policy_ip_block_quota_test.go

```diff
@@ -0,0 +1,254 @@
+// © Broadcom. All Rights Reserved.
+// The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: MPL-2.0
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyIpBlockQuotaCreateAttributes = map[string]string{
+	"display_name":          getAccTestResourceName(),
+	"description":           "terraform created",
+	"single_ip_cidrs":       "2",
+	"ip_block_visibility":   "PRIVATE",
```

Comment:
Nit: quotas are mostly used for external blocks, might be worth focusing on those for testing